### ISSUE TITLE
How to uninstall manually installed pgtap.sql ?

### ISSUE THREAD

**Issue by strk** [theory/pgtap#188]
The uninstall_pgtap.sql script errors out on:
>  psql:uninstall_pgtap.sql:857: ERROR:  syntax error at or near "DROPTYPE"

**Comment by theory** [comment]
That file is generated by `make`. Can you attach it, along with information about your platform, version of `make`, Postgres version, and Perl version?

**Comment by strk** [comment]
[uninstall_pgtap.sql.txt](https://github.com/theory/pgtap/files/2694028/uninstall_pgtap.sql.txt)

Platform is Ubuntu 14.04.5 LTS (on an Amazon EC2 instance)
Perl is v5.18.2
pg_config --version returns PostgreSQL 9.6.11


The line is generating the uninstall_pgtap.sql:
```
grep '^CREATE ' sql/pgtap.sql | '/usr/bin/perl' -e 'for (reverse <STDIN>) { chomp; s/CREATE (OR REPLACE)?/DROP/; print "$_;\n" }' > sql/uninstall_pgtap.sql
```

You are replacing `/CREATE (..)?/` with `/DROP/`
Note there's no space in the replacement, while there is one in the CREATE part.
This only works when the optional part in parens `(OR REPLACE)` is there, because in that case we're NOT replacing any further space.

**Comment by strk** [comment]
It looks like this was already fixed in 024754675b63d5e369d12b268b7caa40763f77ba but no new release was tagged with the fix included. I'm using latest tagged which is 0.99.0

Can we get a new tag/release ?

**Comment by theory** [comment]
Yeah, I'll look at making a release in the next few weeks, unless someone else beats me to it.
